### ISSUE TITLE
Chinese Sign

### ISSUE THREAD

**Issue by Sohair63** [7even/zodiac#20]
Is there any plan to implement Chinese Zodiac Signs in this repo?

**Comment by 7even** [comment]
No, but you can propose a pull request - this feature seems to be pretty easy to implement. I would add it as a method in [Zodiac::Finder](https://github.com/7even/zodiac/blob/master/lib/zodiac/finder.rb) and extend datetime classes with it in [Zodiac::Date](https://github.com/7even/zodiac/blob/master/lib/zodiac/date.rb).

**Comment by Sohair63** [comment]
Thanks, I will research a bit about Chinese Sign and get back to you with this implementation.
